app/assets/stylesheets/blacklight.scss

```diff
@@ -66,3 +66,38 @@
     }
   }
 }
+
+dt.blacklight-full_text_tesimv {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `blacklight-full_text_tesimv` should be written in lowercase with hyphens